test/ExecutionFramework.h

```diff
@@ -68,6 +68,15 @@ class ExecutionFramework
 		std::map<std::string, Address> const& _libraryAddresses = std::map<std::string, Address>()
 	) = 0;
 
+	virtual bytes const& multiSourceCompileAndRunWithoutCheck(
+		std::map<std::string, std::string> const& _sources,
+		u256 const& _value = 0,
+		std::string const& _contractName = "",
+		bytes const& _arguments = bytes(),
+		std::map<std::string, Address> const& _libraryAddresses = std::map<std::string, Address>()
```

Comment:
```suggestion
		bytes const& _arguments = {},
		std::map<std::string, Address> const& _libraryAddresses = {}
```

Same in functions above and below and in `SolidityExecutionFramework.h`.